update.sh

```diff
@@ -12,13 +12,13 @@ VERSION=$1
 echo "Updating substrait submodule..."
 git submodule update --remote third_party/substrait
 
-DIR=$(cd $(dirname $0) && pwd)
-pushd "${DIR}"/third_party/substrait/
-git checkout $VERSION
+DIR=$(cd "$(dirname "$0")" && pwd)
+pushd "${DIR}"/third_party/substrait/ || exit
+git checkout "$VERSION"
 SUBSTRAIT_HASH=$(git rev-parse --short HEAD)
-popd
+popd || exit
 
-VERSION=$(echo $VERSION | sed 's/v//')
+VERSION=${VERSION//v/}
```

Comment:
Does this add a v to the beginning or remove it?

Reply:
it removes it -- bash substitution uses a different collection of slashes as compared to `sed` because of course it does.

`${variable//find/replace}`